tests/progs/test_jubjub.py

```diff
@@ -216,3 +210,22 @@ def test_basic_point_functionality():
             await p1.montgomery_mul(5))
 
     await run_test_prog(_prog, test_preprocessing)
+
+
+@mark.asyncio
+async def test_share_mul(test_preprocessing):
```

Comment:
Can you add another operation to this unit test? I'd like to see that `TEST_POINTS[0] * <some number> == TEST_POINTS[0]` (TEST_POINTS[0] represents zero)